src/script.js

```diff
@@ -83,10 +83,10 @@ $j(document).ready(() => {
 	});
 });
 
-function getGameConfig() {
+export function getGameConfig() {
 	let defaultConfig = {
 		playerMode: $j('input[name="playerMode"]:checked').val() - 0,
-		creaLimitNbr: $j('input[name="activeUnits"]:checked').val() - 0, // DP counts as One
+		creaLimitNbr: $j('input[nlogame="activeUnits"]:checked').val() - 0, // DP counts as One
```

Comment:
it this intentional? looks like a type... `[nlogame=...`

Reply:
It's indeed a typo